src/grid.tsx

```diff
@@ -59,6 +59,181 @@ if ((module as any).hot) {
+ *      @param {number} [options.columnOptions.frozenCount=0] - The number of frozen columns.
+ *          The columns indexed from 0 to this value will always be shown on the left side.
+ *          {@link Grid#setFrozenColumnCount} can be used for setting this value dynamically.
+ *      @param {number} [options.columnOptions.frozenBorderWidth=1] - The value of frozen border width.
+ *          When the frozen columns are created by "frozenCount" option, the frozen border width set.
+ *      @param {Object} [options.treeColumnOptions] - Option object for the tree column.
+ *      @param {string} [options.treeColumnOptions.name] - The name of column that makes tree column.
+ *      @param {boolean} [options.treeColumnOptions.useIcon=true] - If set to true, the folder or file icon is created on
+ *          the left side of the tree cell data.
+ *      @param {boolean} [options.treeColumnOptions.useCascadingCheckbox] - If set to true, a cascading relationship is
+ *          created in the checkbox between parent and child rows.
+ *      @param {Object} [options.copyOptions] - Option object for clipboard copying
+ *      @param {boolean} [options.copyOptions.useFormattedValue] - Whether to use formatted values or original values
+ *          as a string to be copied to the clipboard
+ *      @param {boolean} [options.useClientSort=true] - If set to true, sorting will be executed by client itself
+ *          without server.
+ *      @param {string} [options.editingEvent='dblclick'] - If set to 'click', editable cell in the view-mode will be
+ *          changed to edit-mode by a single click.
+ *      @param {boolean} [options.scrollX=true] - Specifies whether to show horizontal scrollbar.
+ *      @param {boolean} [options.scrollY=true] - Specifies whether to show vertical scrollbar.
+ *      @param {boolean} [options.showDummyRows=false] - If set to true, empty area will be filled with dummy rows.
+ *      @param {?string} [options.keyColumnName=null] - The name of the column to be used to identify each rows.
+ *          If not specified, unique value for each rows will be created internally.
+ *      @param {boolean} [options.heightResizable=false] - If set to true, a handle for resizing height will be shown.
+ *      @param {Object} [options.pagination=null] - Options for tui.Pagination.
+ *          If set to null or false, pagination will not be used.
+ *      @param {string} [options.selectionUnit='cell'] - The unit of selection on Grid. ('cell', 'row')
+ *      @param {Array} [options.rowHeaders] - Options for making the row header. The row header content is number of
+ *          each row or input element. The value of each item is enable to set string type. (ex: ['rowNum', 'checkbox'])
+ *          @param {string} [options.rowHeaders.type] - The type of the row header. ('rowNum', 'checkbox', 'radio')
+ *          @param {string} [options.rowHeaders.title] - The title of the row header on the grid header area.
```

Comment:
@param {string} [options.rowHeaders.header] - The header title of the row header on the grid header area.